protobuf/datastore.proto

```diff
@@ -99,6 +99,10 @@ message TSMetadata {
   string unit = 21;
   string instrument = 22;
   string instrument_vocabulary = 23 [json_name = "instrument_vocabulary"];
+  string level = 24;
+  string period = 25;
+  string function = 26;
+  string parameter_name = 27;
```

Comment:
Why do we want to ingestor to provide this? It should be constructed from the other four, won't it just violate the single source of truth principle?

To me this is just an EDR API level construct, which can/should be constructed on the fly.